test/cudafortran.scm

```diff
@@ -0,0 +1,34 @@
+;;; Functions (with attributes)
```

Comment:
What is this .scm file?

Reply:
Sorry, it was for local consumption, pushed it by mistake. I have removed it now.